container_images/test-manager/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ValidateReturn() {
+  RET=$?
+  if [ $RET -ne 0 ]; then
+    GOBUILD_OUT=$RET
+    echo "'go build or go test -c' exited with ${GOBUILD_OUT}"
+  else
+    echo "'go build or go test -c' success"
+  fi
+}
+
+export CGO_ENABLED=0 GOOS=linux
+BUILD_DIR=./test_manager
+chmod o+x -R $BUILD_DIR
+[ -n "$BUILD_DIR" ] && cd $BUILD_DIR
+mkdir /out
+
+GOBUILD_OUT=0
+
+echo "Start Building"
+for d in ./*; do
+  d=`basename $d`
+  if [ "${d}" = "test_suites" ];then
```

Comment:
bash style says use [[ ]] tests and a space after the semicolon

Reply:
also i dont like this testing logic. you know you're going to handle specific directories, let's just list them and treat them separately. one block for building the test packages, one for building the tools

Reply:
I split two for loop, one is for build test manager and test wrapper, another is for build test packages.

Reply:
no for loops
go into test suite dir, and build all those
then go to test manager and build it
then go to test wrapper and build it
no for loops

Reply:
Ok. Got it.